css/jspage.css

```
@@ -0,0 +1,200 @@
+
+}
+
+.divhead{
+    font-size: 1rem;
+    margin: 1.875rem 0rem;
+    padding: 0 0.625rem;
+  
+    border-left: medium outset #efff15;
+    border-radius: 4px;
+    color: #ebf82c;
+    font-size: 43px;
```

Biome (2.1.2)

[error] Duplicate properties can lead to unexpected behavior and may override previous declarations unintentionally.

font-size is already defined here.

Remove or rename the duplicate property to ensure consistent styling.

(lint/suspicious/noDuplicateProperties)